client/src/pages/Settings/Availability/Availability.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { Fragment, useState } from 'react';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+import { Box } from '@mui/system';
+import { ListItemText, Typography } from '@mui/material';
+import useStyles from './useStyles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableRow from '@mui/material/TableRow';
+import TextField from '@mui/material/TextField';
+import Checkbox from '@mui/material/Checkbox';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import 'date-fns';
```

Comment:
Importing just `date-fns` isn't really doing anything for us 

import dateFns from 'date-fns'`

Also, this should be moved to the top of the file for organization.